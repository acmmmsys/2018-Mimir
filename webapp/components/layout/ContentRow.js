import React, { Component } from 'react';
import { Grid } from 'semantic-ui-react'

import 'style/layout/GridLayout.scss';

class ContentRow extends Component {
    render()Â {
        return (
            <Grid.Row className='grid-row'>
                {this.props.children}
            </Grid.Row>
        );
    }
}

export default ContentRow;