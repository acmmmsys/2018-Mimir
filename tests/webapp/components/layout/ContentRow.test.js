import React from 'react';
import ContentRow from '../../../../webapp/components/layout/ContentRow'
import renderer from 'react-test-renderer';

let wrapper;

test('ContentRow does render', () => {
    wrapper = shallow(
        <ContentRow>

        </ContentRow>
    );
    expect(wrapper).toMatchSnapshot();
});
