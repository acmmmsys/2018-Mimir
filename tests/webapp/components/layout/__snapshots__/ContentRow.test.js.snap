// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentRow does render 1`] = `
<GridRow
  className="grid-row"
/>
`;
